test/tstBoostGeometryAdapters.cpp

```diff
@@ -75,8 +75,8 @@ BOOST_AUTO_TEST_CASE(distance)
 
   std::tie(a, b) = std::make_pair<details::Point, details::Point>(
       {{0., 0., 0.}}, {{1., 1., 1.}});
-  BOOST_TEST(details::distance(a, b) == std::sqrt(3.));
-  BOOST_TEST(bg::distance(a, b) == std::sqrt(3.));
+  BOOST_TEST(details::distance(a, b) == std::sqrt(3.f));
+  BOOST_TEST(bg::distance(a, b) == std::sqrt(3.0));
```

Comment:
Make sure you see that as well.  Boost promote the distance return type to double.